sdk.go

```diff
@@ -271,3 +271,36 @@ func (*Sdk) Events(start, end string) ([]map[string]interface{}, error) {
 
 	return events, nil
 }
+
+// UserInfo contains user info returned by daemon.
+type UserInfo struct {
+	ID       string `json:"id"`
+	Username string `json:"username"`
+	Email    string `json:"email"`
+}
+
+// User returns the user info for the current user running the Op.
+func (*Sdk) User() (UserInfo, error) {
+	var body interface{}
+	method := "GET"
+	result, err := daemon.SyncRequest("user", body, method)
+
+	if err != nil {
+		return UserInfo{}, fmt.Errorf("error getting user information: %w", err)
+	}
+
+	// map results to UserInfo
+	mapValue := result.(map[string]interface{})
+	userInfo := UserInfo{}
+	if id, ok := mapValue["id"].(string); ok {
+		userInfo.ID = id
+	}
+	if username, ok := mapValue["username"].(string); ok {
+		userInfo.Username = username
+	}
+	if email, ok := mapValue["email"].(string); ok {
+		userInfo.Email = email
+	}
```

Comment:
```suggestion
	userInfo := UserInfo{}
	err := json.Unmarshal(result, &userInfo)
```
Should be able to automatically unmarshal the JSON into the truct

Reply:
I can try that, the issue is that the `result` payload has more fields than `UserInfo` so I think it will throw an error.

Reply:
@mattcan I tried it and I cannot do it because `result` is of type interface{} (stringified JSON) it is not of type `[]bytes`. So it has to stay as it is